packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -2719,12 +2719,18 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
             return undefined;
           } catch (error) {
             assertErrorThrown(error);
-            const result = new BulkRegistrationResultFail(error.message);
+            let result = new BulkRegistrationResultFail(error.message);
+
             if (graphData instanceof V1_PureModelContextData) {
-              result.servicePath = getNullableFirstElement(
+              const servicePath = getNullableFirstElement(
```

Comment:
it might be better to just create different error.message(string) instead of new BulkRegistrationResultFail twice